src/ingest_validation_tools/table-schemas/assays/ims-v2.yaml

```diff
@@ -0,0 +1,132 @@
+doc_url: https://portal.hubmapconsortium.org/docs/assays/maldi-ims
+fields:
+# include: ../includes/vA/all_assays.yaml
+- name: assay_category
+  constraints:
+    enum:
+      - mass_spectrometry_imaging # TODO: This is an internal ID; would prefer not to change.
+- name: assay_type
+  constraints:
+    enum:
+      - MALDI-IMS
+      - SIMS-IMS
+      # TODO: "NanoDESI" is already a value under the nano schema: It can't be in both.
+      - DESI
+- name: analyte_class
+  constraints:
+    enum:
+      - protein # NOTE: "proteins" specified, but singular is used elsewhere.
+      - metabolites
+      - lipids
+      - peptides
+      - phosphopeptides
+      - glycans
+
+# include: ../includes/v0/is_targeted.yaml
+
+# Level 2:
+
+# include: ../includes/v0/acquisition_instrument.yaml
+- name: ms_source
+  description: The ion source type used for surface sampling (MALDI, MALDI-2, DESI, or SIMS) or LC-MS/MS data acquisition (nESI)
+  constraints:
+    enum:
+      - MALDI
+      - MALDI-2
+      - LDI
+      - LA
+      - SIMS-C60
+      - SIMS-H2O
+      - ESI
+      # TODO: If any data was uploaded with deprecated value, what should it map to?
+      # - DESI
+      # - SIMS
+      # - nESI
+- name: polarity # TODO: Why is this green in the spreadsheet?
+  description: The polarity of the mass analysis (positive or negative ion modes)
+  constraints:
+    enum:
+      - negative ion mode
+      - positive ion mode
+- name: mz_range_low_value # TODO: Why is this green in the spreadsheet?
+  description: The low value of the scanned mass range for MS1. (unitless)
+- name: mz_range_high_value # TODO: Why is this green in the spreadsheet?
+  description: The high value of the scanned mass range for MS1. (unitless)
+
+- name: mass_resolving_power
+  description: This is the MS1 resolving power. This is a unitless value often calculated as m/∆m where ∆m is the FWHM for a given peak with a certain m/z (m).
+  type: number
+- name: mz_resolving_power
+  description: This is the m/z used to calculate the resolving power.
+
+- name: ion_mobility
+  description: 'Methodologies for measuring ion mobility: Traveling Wave Ion Mobility Spectrometry (TWIMS), Trapped Ion Mobility Spectrometry (TIMS), High Field Asymmetric waveform ion Mobility Spectrometry (FAIMS), Drift Tube Ion Mobility Spectrometry (DTIMS), Structures for Lossless Ion Manipulations Spectrometry (SLIMS).'
+  constraints:
+    required: False
+    enum:
+      - TIMS
+      - TWIMS
+      - FAIMS
+      - DTIMS
+      - SLIMS # TODO: Confirm that adding final "S" here is correct?
+- name: ms_scan_mode
+  description: TODO
```

Comment:
```suggestion
  description: Scan mode refers to the number of steps in the separation of fragments.
```

Reply:
@cebriggs7135 -- This will be an in integer? Seems strange to me.